samples/radio_test/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8.2)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
```

Comment:
According to #1680, the sample should also include nrf's boilerplate (`include($ENV{ZEPHYR_BASE}/../nrf/cmake/boilerplate.cmake)`)